hybrasyl/Connection.cs

```diff
@@ -69,25 +69,26 @@ public static void DeregisterClient(Client client)
             {
                 ((IDictionary)WorldClients).Remove(client.ConnectionId);
                 // This will also handle removing the user from WorldClients if necessary
-                try
-                {
-                    World.ControlMessageQueue.Add(new HybrasylControlMessage(ControlOpcodes.CleanupUser, client.ConnectionId));
-                }
-                catch (InvalidOperationException)
-                {
-                    if (!World.ControlMessageQueue.IsCompleted)
-                        GameLog.ErrorFormat("Connection {id}: DeregisterClient failed", client.ConnectionId);
-                }
+                World.ControlMessageQueue.Add(new HybrasylControlMessage(ControlOpcodes.CleanupUser, client.ConnectionId));

```

Comment:
this needs to be put back, otherwise you get unhandled exceptions / errors on server shutdown